crates/uv/src/commands/venv.rs

```diff
@@ -188,5 +189,22 @@ async fn venv_impl(
         }
     }
 
+    match &platform.os() {
+        Os::Windows => {
+            writeln!(
+                printer,
+                "Activate with .\\{}\\Scripts\\activate.ps1",
+                path.normalized_display().cyan()
+            )
+        },
+        _ => {
+            writeln!(
+                printer,
+                "Activate with source {}/bin/activate",
```

Comment:
```suggestion
                "Activate with: source {}/bin/activate",
```
or put command in quotations for improve readability.

Reply:
Good suggestion, have added this in.